ui/templates/submission_create.html

```diff
@@ -7,26 +7,26 @@ <h1 class="section__title">Submit New Brain</h1>
   <div class="section__body typo">
     <blockquote class="warn">
       <ul>
-        <li class="text-red">Do not copy others' source code. We will run plagiarism detection programs to compare your source code. Copycats will fail this project!</li>
+        <li class="text-red">Do not copy others' source code. All the code you submit will be checked for plagiarism. Copycats will fail this project!</li>
         <li>Do not share your source code with others. Similar submissions will be treated as plagiarism.</li>
-        <li>Do not use alternative account to submit your code, otherwise yourself will be recognized as a copycat.</li>
+        <li>Do not use alternative account to submit your code, otherwise you will be recognized as a copycat.</li>
         <li>Do not submit C++ source code. Only C language is supported.</li>
-        <li>Do not abuse our cloud execution service, for example, intentionally trying to shutdown our judge machine.</li>
+        <li>Do not abuse our cloud execution service, such as intentionally trying to shutdown our judge machine.</li>
       </ul>
     </blockquote>
     <blockquote class="note">
       <ul>
-        <li>We recommended you to use <a href="https://sourceforge.net/projects/orwelldevcpp/" target="_blank">Dev-C++</a> instead of Visual Studio.</li>
```

Comment:
it is worse than the original for ambiguities. Besides, "officially recommended" is not common.